docs/userGuide/cliCommands.md

```diff
@@ -161,11 +161,10 @@ The caveat is that not building all pages during the initial process, or not reb
   {{ icon_example }} `-s otherSite.json`
```

Comment:
Perhaps we could fix a typo (give -> given) in line 156 while we are editing this section!

Reply:
Good catch! Fixed in [2e73648](https://github.com/MarkBind/markbind/pull/2247/commits/2e73648dee577fcdc954b735e17e231648094dcb).